### PR TITLE
Save and reload login access token to/from disk by default.

### PR DESCRIPTION
**Pull request by rnd256**
This speeds up the login process, and is especially useful when you're
writing/testing code and need to keep restarting the bot.

It also makes bots less impacted by the frequent login flakiness
(http://ispokemongodownornot.com/).

### CODE REVIEW

File: src/Player.js [hunk 1]
```diff
@@ -82,8 +86,48 @@ class Player {
...
   }
 
-  async Login(user, pass) {
-    let res = await this.Auth.login(user, pass, this.playerInfo.provider)
+  async Login(user, pass, forceRefreshLogin) {
+    var loginCacheString = null
+    var loginCache = null
+
+    if (!forceRefreshLogin) {
+      console.log('Checking for login cache.')
```
Comment (stoffern) [inline src/Player.js]: use `this.parent.log.info(`log text`)` not `console.log`

**Comment by stoffern** [comment]
Thanks for PR. I think it should be defaut login with username and password.
And if they are not provided, then search for token.

**Comment by stoffern** [comment]
@jimmywarting any feedback on this?

**Comment by jimmywarting** [comment]
lgtm

**Pull request merged**